proto/viam/app/v1/app.proto

```diff
@@ -122,10 +153,96 @@ message Organization {
   google.protobuf.Timestamp created_on = 3;
 }
 
+message OrganizationMember {
+  string user_id = 1;
```

Comment:
hmm. What does `user_id` mean in this case?  Is this our internal ID? or the email(s)?

Reply:
This is the member's userId, not their email.

Reply:
Wont we need the email(s) in the frontend?

Reply:
Indeed, will update

Reply:
updated